script/dsl-scripts/KIE_Release.sh

```diff
@@ -0,0 +1,628 @@
+def releasePrepare =
+"""
+sh /home/jenkins/workspace/DSL-KIE-Release-6.5.x/script/release/01.KIE_releasePrepare.sh
```

Comment:
We really should not use hardcoded paths here. It makes the scripts hard to use on different environments (e.g. locally when testing the changes). Could we just use the relative path `script/release/<name>`?